atom/browser/lib/guest-window-manager.coffee

```diff
@@ -30,6 +32,10 @@ createGuest = (embedder, url, frameName, options) ->
     guest.once 'closed', ->
       delete frameToGuest[frameName]
 
+  ipc.on 'ATOM_SHELL_GUEST_WINDOW_MANAGER_WINDOW_OPENER_POSTMESSAGE', (event, method, args...) ->
```

Comment:
The `method` here is not used, we should remove it from parameters.

And since we are not using variadic arguments here, we should use `message, targetOrigin` directly instead of using `args...` and referencing them as `args[0]` and `args[1]`.

So the final parameters should be `(event, message, targetOrigin)`.